src/Schema/Events/Conversations.ts

```diff
@@ -2,6 +2,72 @@ import { ContextModule } from "../Values/ContextModule"
 import { OwnerType } from "../Values/OwnerType"
 import { ActionType } from "."
 
+/**
+ * A user taps or clicks to accept an offer
+ *
+ * This schema describes events sent to Segment from [[acceptOffer]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "acceptOffer",
+ *    context_owner_type: "inboxConversation"
+ *    impulse_conversation_id: "198"
+ *    subject: "Confirm & Pay"
+ *  }
```

Comment:
[Commented](24c25e5a-ec98-466e-864d-c3dd9962a27b) on the spreadsheet that we decided to reuse the webview so we won't be building the review offer screen. We can probably ignore the `AcceptOffer`, `CounterOffer` and `DeclineOffer` here if we already have tracking for the existing flow.

Reply:
Not add add more work but I wonder if tracking tapping on the order CTA would be worthwhile, such as the green "Offer Accepted" bar.

![Screen Shot 2021-03-05 at 2 56 02 PM](https://user-images.githubusercontent.com/796573/110166907-fd326d80-7dc2-11eb-84fe-c88e0021fb53.png)

Reply:
Yes we want to track that with the `tappedViewOffer` event using the `subject` field to figure out the status that they are tapping on. Does that make sense?